### PR TITLE
Cran release

### PR DESCRIPTION
**Pull request by elimillera**
Hey @shengwei66 @nicholas-masel ,

Here are the updates to get windows working. It was mainly just some warnings that only surfaced with windows and some path normalization. 

The rhub check has cleared for windows and I have it running for macos as well.

### CODE REVIEW

**Comment by elimillera** [comment]
Looking good on macos

File: tests/testthat/test-getFileName.R [hunk 1]
```diff
@@ -4,6 +4,8 @@ test_that("getFileNameBatch returns the expected results", {
   cmdArgs2 <- c("--arg1", "someThing", "-e", 'timber::axecute(file="/asdf/programs/CopyOftest_demo.R",log_name="CopyOftest_demo.R.log",log_path="/adr/PDEV/emiller9/pharma/test_compound/mi_test/dbr_tidytlg/re_test/output")')
```
Comment (shengwei66) [inline tests/testthat/test-getFileName.R:4]: @elimillera the log_path argument in cmdArgs2 is using the J&J file structure. Can we change this argument to be more general?

**Review by shengwei66 — COMMENTED** [review]
@elimillera these commits look good. I just have a minor comment on the log_path in the test-getFileName.R (please see my comment there).

**Review by shengwei66 — APPROVED** [review]
these changes are good!

**Pull request merged**